examples/features/advancedtls/generate.sh

```diff
@@ -0,0 +1,87 @@
+rm -rf creds
```

Comment:
A little scary to me when I see `rm -rf` in a script.  You could either test for existence of the directory and refuse to run or prompt for confirmation or something.